addons/website/views/snippets/snippets.xml

```diff
@@ -278,7 +278,12 @@
     <div data-js="background" string="Background"
         data-selector="section, .parallax, .carousel-item"
         data-exclude=".s_hr, .s_image_gallery, .o_gallery .carousel-item">
-        <we-imagepicker string="Background" data-background="" data-allow-videos=":not(.parallax, .s_parallax_bg)"/>
+        <we-imagepicker string="Background" data-background="" data-allow-videos=":not(.parallax, .s_parallax_bg)" data-name="bg_img_opt"/>
+    </div>
+    <div data-js="BackgroundOptimize" string="Background"
+        data-selector="section, .parallax, .carousel-item"
+        data-exclude=".s_hr, .s_image_gallery, .o_gallery .carousel-item">
+        <t t-call="web_editor.image_optimization_controls"/>
```

Comment:
Instead of duplicating the data-selector and data-exclude, share them in a variable